packages/snack-sdk/src/transports/index.ts

```diff
@@ -1,8 +1,11 @@
 import CompositedTransport from './CompositedTransport';
+import ConnectionMetricsEmitter from './ConnectionMetricsEmitter';
 import TransportImplPubNub from './TransportImplPubNub';
 import { SnackTransport, SnackTransportOptions } from './types';
 
 export * from './types';
+export * from './ConnectionMetricsEmitter';
+export { ConnectionMetricsEmitter };
```

Comment:
Any reason to not export it from the file directly?

```suggestion
export { ConnectionMetricsEmitter } from './ConnectionMetricsEmitter';
```